css/style.css

```diff
@@ -0,0 +1,6 @@
+img{
```

Comment:
Great job mentioning this to your student but I suggest that you phrase your comment as a suggestion, be polite,  and give a reason for the requested change.

### EXAMPLE
I kindly suggest that you should a space between tag selector and { because it will make your CSS neater